<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setup</class>
 <widget class="QDialog" name="setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="votefor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>421</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Title for Voting System</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="candidate_1_img">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Candidate 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QLabel" name="candidate_2_img">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Candidate 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Candidate 1 Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Candidate 2 Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="connect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[+]</string>
   </property>
  </widget>
  <widget class="QComboBox" name="address">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Chitwan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kathmandu</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Location:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
